onnx_chainer/export.py

```diff
@@ -48,32 +48,86 @@ def convert_parameter(parameter, context):
     return numpy_helper.from_array(array, context.get_name(parameter))
 
 
-def rename_tensors(model):
+def rename_tensors(model, force_rename_network_out=True):
     names = {v.name: v.name for v in model.graph.initializer}
+    network_output_names = set() if force_rename_network_out else\
+        {o.name for o in model.graph.output}
     op_counts = collections.defaultdict(int)
 
     for op in model.graph.node:
+        for i, input_name in enumerate(op.input):
+            if input_name not in names:
+                names[input_name] = input_name
+            op.input[i] = names[input_name]
+
         op_name = '{}_{}'.format(op.op_type, op_counts[op.op_type])
         op_counts[op.op_type] += 1
-
-        for i in range(len(op.input)):
-            if op.input[i] not in names:
-                names[op.input[i]] = 'Input_{}'.format(op_counts['Input'])
-                op_counts['Input'] += 1
-            op.input[i] = names[op.input[i]]
-
-        for i in range(len(op.output)):
-            if len(op.output) <= 1:
-                names[op.output[i]] = op_name
-            else:
-                names[op.output[i]] = '{}_{}'.format(op_name, i)
-            op.output[i] = names[op.output[i]]
+        if len(op.output) == 1:
```

Comment:
[optional] I feel the original structure was easier to read (branch by `len(op.output)` in the for loop over `enumerate(op.output)`).